libs/langgraph/langgraph/pregel/algo.py

```diff
@@ -46,6 +46,8 @@
 from langgraph.pregel.types import All, PregelExecutableTask, PregelTask
 from langgraph.utils.config import merge_configs, patch_config
 
+EMPTY_SEQ = tuple()
```

Comment:
Do you have an idea of how helpful this was? I guess this is sort of an inner loop

Reply:
This was more because the previous code was actually incorrect, as it could end up trying to do `<str> in None`